sagemaker/launch/common_parser.py

```diff
@@ -39,9 +39,23 @@ def create_sm_session(instance_type, region):
     """
     if instance_type == "local":
         sess = LocalSession()
+        # Add local SM config if needed
         if sess.config is None:
             sess.config = {}
-        sess.config.update({'local': {'local_code': True}})
+        sess.config.update({"local": {"local_code": True}})
+
+        # Set shm_size if needed
+        should_set_shm = (
```

Comment:
Please add a comment:
\# if shm_size is not specified, use 90% of host memory as shared memory.